docs/metrics/bleu.md

```diff
@@ -0,0 +1,123 @@
+# BLEU (**B**i**L**ingual **E**valuation **U**nderstudy) Score
+
+The BLEU score is a metric commonly used in a variety of NLP tasks ranging from Machine Translation to Text Summarization, to evaluate the quality of machine-generated text. It quantifies the similarity between the generated text and ground truth text as a score between 0 and 1 — 0 meaning there is no overlap with the ground truth text, and 1 meaning there is a perfect overlap. As the name suggests, it was originally developed for evaluating machine-translation models, but has since been adapted to many different tasks within NLP due to its dynamic nature for measuring textual similarity.
+
+## Implementation Details
```

Comment:
Before jumping into the `Definition` section, I'd include a quick numbered list here describing the components of BLEU: n-gram overlap, and brevity penalty. Give readers a gentle introduction before hitting them with the formula

Reply:
bump